Add how to clean up local env.

KMiller-Grafana opened the pull request:
Added only to the Grafana Fundamentals tutorial.

matthewhelmke reviewed (approved):
LGTM

oddlittlebird commented on content/tutorials/grafana-fundamentals.md:
```diff
@@ -345,6 +345,15 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
 
+### Clean up the local environment
+
+The tutorial has running Docker containers.
```
Please remove whatever is auto-ending your lines. It's a real pain for reviewing.

oddlittlebird commented on content/tutorials/grafana-fundamentals.md:
```diff
@@ -345,6 +345,15 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
+The tutorial has running Docker containers.
+When you want to clean up this local environment such that there are no
+running Docker containers,
+use `docker ps` to acquire the IDs of the tutorial's containers.
```
```suggestion
use `docker ps` to acquire the IDs of the tutorial containers.
```

matthewhelmke replied:
Is this a Grafana style thing, @oddlittlebird ? Because the way all these instances were originally is a very normal way to refer to containers like these. The possessive form isn't strictly necessary.

osg-grafana replied:
Are you two suggesting the exact same thing? :)

oddlittlebird commented on content/tutorials/grafana-fundamentals.md:
```diff
@@ -345,6 +345,15 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
+running Docker containers,
+use `docker ps` to acquire the IDs of the tutorial's containers.
+Stop and remove each tutorial container with `docker kill` *`CONTAINER_ID`*.
+Replace *`CONTAINER_ID`* with the container's ID.
```
```suggestion
Replace *`CONTAINER_ID`* with the container ID.
```

oddlittlebird commented on content/tutorials/grafana-fundamentals.md:
```diff
@@ -345,6 +345,15 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
+When you want to clean up this local environment such that there are no
+running Docker containers,
+use `docker ps` to acquire the IDs of the tutorial's containers.
+Stop and remove each tutorial container with `docker kill` *`CONTAINER_ID`*.
```
Where do you do this? The command line? If so, which one? Git Bash? cmd? PowerShell? If PowerShell, do I need to Run as admin?

osg-grafana replied:
I would assume any terminal will do because Docker is cross platform.

oddlittlebird reviewed (changes requested):
A few suggestions and a question.

marcusolsson reviewed (changes requested):
The tutorial environment is using Docker Compose, so you should be able to run `docker-compose down -v` to clean up. The `-v` purges any temporary volumes in addition to any running containers.

achatterjee-grafana commented:
Should we refer users to docker documentation on how to clean up the local environment?

KMiller-Grafana commented:
@oddlittlebird @marcusolsson Please re-review.  Thanks for the comments. The instructions are simple now.

oddlittlebird commented on content/tutorials/grafana-fundamentals.md:
```diff
@@ -345,6 +345,14 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
 
+### Clean up the local environment
+
+The tutorial has running Docker containers. When you want to clean up this local tutorial environment, run
```
Still not telling me where to run this. Since the previous steps are in Grafana, we need to let the user know that they are shifting to a terminal of some sort.

KMiller-Grafana replied:
Unless you want me to revise the entire tutorial with this PR, this additional instruction follows what has been assumed in the earlier parts of the tutorial, where there were other docker-compose commands. My thinking:  if a customer already got the earlier tutorial steps working, they know how/where to run a Docker command.

matthewhelmke replied:
This is really fine, unless you are volunteering to edit the entire piece. This was intended as a simple edit. The intended audience knows how to use Docker. This isn't a total beginner-focused tutorial.

osg-grafana commented on content/tutorials/grafana-fundamentals.md:
```diff
@@ -345,6 +345,14 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
 
+### Clean up the local environment
+
+The tutorial has running Docker containers. When you want to clean up this local tutorial environment, run
```
```suggestion
The tutorial has running Docker containers. When you want to clean up this local tutorial environment, open a terminal and run the following command:
```

marcusolsson commented:
Looks good to me!

KMiller-Grafana commented:
With both approvals and unaddressed issues on this PR, I'm merging it in now and calling this task completed.  I've created issue https://github.com/grafana/tutorials/issues/122 to address the "where do I type in my commands?" issue. I've also created issue https://github.com/grafana/tutorials/issues/121 to propagate environment cleanup details into the other tutorials.
